### PR TITLE
Add comments in api-client module

### PR DESCRIPTION
**Pull request by patuwwy**

### CODE REVIEW

File: packages/api-client/src/host-client.ts [hunk 1]
```diff
@@ -45,22 +82,52 @@ export class HostClient implements ClientProvider {
...
+     * Returns Host version.
+     *
+     * @returns {Promise<Response>} Promise resolving to Host version.
+     */
     async getVersion() {
         return this.client.get("version");
     }
 
+    /**
+     *
```
Comment (alicja-gruzdz) [inline packages/api-client/src/host-client.ts:114]: Maybe?
```suggestion 
     * Send data to the topic. 
     * Topics are a part of Service Discovery feature enabling data exchange through Topics API.
     *
 ```

File: packages/api-client/src/instance-client.ts [hunk 2]
```diff
@@ -52,6 +75,12 @@ export class InstanceClient {
         );
     }
 
+    /**
+     *
```
Comment (alicja-gruzdz) [inline packages/api-client/src/instance-client.ts:79]:
```suggestion
     * Send event to the Instance.
     *
  ```

**Review by alicja-gruzdz — APPROVED** [review]

File: packages/api-client/src/host-client.ts (hunk 1 above)
Comment (a-tylenda) [reply packages/api-client/src/host-client.ts:114]: Sends

File: packages/api-client/src/host-client.ts [hunk 3]
```diff
@@ -13,28 +17,61 @@ export class HostClient implements ClientProvider {
...
+     * Returns Host log stream.
+     *
+     * @returns {Promise<Response>} Promise resolving to response with log stream.
+     */
     async getLogStream() {
         return this.client.getStream("log");
     }
 
+    /**
+     * Upload sequence to Host.
```
Comment (patuwwy) [inline packages/api-client/src/host-client.ts:48]:
```suggestion
     * Uploads sequence to Host.
```

**Pull request merged**